ProjectSidewalk/SidewalkWebpage
# 1426 next city no sidewalk label pop up

Partial resolves #1426.

**Summary of changes:**
1. "No Sidewalk" label produces a context menu
2. "No Sidewalk" label takes a severity rating
3. "No Sidewalk" label has tags

**To test:**
- Go to audit
- Add "No Sidewalk" label
- Check that the context menu comes up, along with tags & severity ratings
- Check that the hotkeys for the tags work, and that mutually exclusive options are mutually exclusive
- Check that any tags and severity ratings submitted are saved in the DB

**Note that next steps to fully resolve this PR are required:**
1. we will need to amend the tutorial to accommodate this. I took a look but don't think it is an easy change and would like to focus more on mobile.
2. we will also need to take these same steps for the occlusion label

**Callout:**
I haven't gotten this recently, but want to make sure that if it is an issue, we catch it early. the way I've written the evolutions file deletes everything from the "tag" table where label_id = 7 (no sidewalk) on the downs, however it does not delete any labelled tags from the table "label_tag." For a while, this would cause "DB in inconsistent state" errors., However these are not showing anymore and I don't think I should be deleting anything from "label_tag." Let me know if this is a problem though and we can discuss. I'm not super familiar with evolutions so want to make sure all is ok.
